src/components/organisms/AdminVenueView/components/Timing/Timing.tsx

```diff
@@ -48,8 +52,8 @@ export const Timing: React.FC<TimingProps> = ({
         worldId: venue?.worldId,
       },
       user
-    );
-  }, [venue, user, startUtcSeconds, endUtcSeconds]);
+    ).catch((e) => console.error(Timing.name, e));
```

Comment:
Why do we need this error handling?

Reply:
Because it's to better at least write it some place (even in console, for dev purpose) than to ignore/hide errors.